Documentation/gettingstarted/policy-creation.rst

```diff
@@ -0,0 +1,81 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _policy_verdicts:
+
+*******************************
+Creating policies from verdicts
+*******************************
+
+Policy Audit mode allows the user allow all traffic to pass through Cilium
+while logging all connections that would otherwise be dropped by policy using
+policy verdict notifications. Policy audit mode may be configured for the
+entire daemon using ``--policy-audit-mode=true``. Layer 3 and 4 policies are
+currently supported. When policy audit mode is enabled and you've installed a
+policy that doesn't match traffic, ``cilium monitor`` prints the following:
+
+.. code-block:: shell-session
+
+   # cilium monitor -t policy-verdict
+   ...
+   Policy verdict log: flow 0x63113709 local EP ID 1121, remote ID 2986, dst port 80, proto 6, ingress true, action audit, match none, 10.29.210.187:54134 -> 10.29.50.40:80 tcp SYN
+
+In the above example, we can see that traffic to endpoint ``11211`` has
+received traffic (``ingress true``) which doesn't match the policy (``action
```

Comment:
is there a corresponding egress policy verdict notification in the node where remote is running?

Reply:
Yes:

```
Policy verdict log: flow 0x1c39ef18 local EP ID 969, remote ID 22111, dst port 80, proto 6, ingress false, action audit, match none, 10.29.112.245:60218 -> 10.29.179.216:80 tcp SYN
Policy verdict log: flow 0x1c39ef18 local EP ID 2239, remote ID 17138, dst port 80, proto 6, ingress true, action allow, match L3-L4, 10.29.112.245:60218 -> 10.29.179.216:80 tcp SYN
```

One trick I found while reviewing your feedback is that there needs to be a policy already for audit mode to take effect. If policy is not enabled, there won't be policy notifications AFAICT.